besu/src/main/java/org/hyperledger/besu/Runner.java

```diff
@@ -108,6 +112,7 @@ public class Runner implements AutoCloseable {
       final Optional<GraphQLHttpService> graphQLHttp,
       final Optional<WebSocketService> webSocketRpc,
       final Optional<JsonRpcIpcService> ipcJsonRpc,
+      final Map<String, JsonRpcMethod> inProcessRpcMethods,
```

Comment:
Comparing to JsonRpcHttpService, I think in-process is missing metrics, e.g. requestTimer. 

Wonder If it's still useful to have separate metrics for these in-process RPC calls?

Reply:
good points, I think it is useful to have metrics, will review this in a separate PR